global.json

```diff
@@ -1,13 +1,13 @@
 {
   "sdk": {
     "version": "3.0.100-preview-009750"
-  },
+  },	   
```

Comment:
This file only has whitespace changes, should remove from PR

Reply:
Sounds good. Thanks for catching.